playbooks/roles/cluster/tasks/main.yml

```diff
@@ -1,2 +1,33 @@
 ---
+- hosts: localhost
```

Comment:
Should this be in a task named bootstrap rather than in main? I'd imagine there will be a number of things we'll want to do in the cluster role so would be good to categorize the tasks

Reply:
Of course yep you are right, I've added a bootstrap.yml which includes the bootstrap_ssh_keys.yml